starknet_py/net/full_node_client.py

```diff
@@ -93,6 +95,47 @@ async def get_block_traces(
     ) -> BlockTransactionTraces:
         raise NotImplementedError()
 
+    async def get_events(
+        self,
+        from_block_number: Optional[Union[int, Tag]] = None,
+        from_block_hash: Optional[Union[Hash, Tag]] = None,
+        to_block_number: Optional[Union[int, Tag]] = None,
+        to_block_hash: Optional[Union[Hash, Tag]] = None,
+        address: Optional[Hash] = None,
+        keys: Optional[List[Hash]] = None,
+        continuation: bool = False,
```

Comment:
Let's rename it to something more descriptive. Perhaps `follow_continuation_token`?

cc @war-in @drknzz

Reply:
`follow_continuation_token` is fine, let's rename it